docs/content/development/extensions-core/avro.md

```diff
@@ -4,7 +4,7 @@ layout: doc_page
 
 # Avro
 
-This extension enables Druid to ingest and understand the Apache Avro data format. Make sure to [include](../../operations/including-extensions.html) `druid-avro-extensions` as an extension.
+This extension enables Druid to ingest and understand the Apache Avro data format. Make sure to [include](../../operations/including-extensions.html) `druid-avro-extensions` as an extension.  Inline schema and [Schema Repo](http://schemarepo.org) are supported.
```

Comment:
actually now we support confluent schema registry as well. I wouldn't mention the list here as that is prone to changes.